components/territory-header.js

```diff
@@ -19,6 +19,7 @@ export function TerritoryDetails ({ sub }) {
           territory details
           {sub.status === 'STOPPED' && <Badge className='ms-2' bg='danger'>archived</Badge>}
           {(sub.moderated || sub.moderatedCount > 0) && <Badge className='ms-2' bg='secondary'>moderated{sub.moderatedCount > 0 && ` ${sub.moderatedCount}`}</Badge>}
+          {(sub.nsfw) && <Badge className='ms-2' bg='secondary'>nsfw</Badge>}
```

Comment:
i used the same styles as the "moderated" badge. lmk if we should use a different color